src/telliot_feeds/cli/commands/report.py

```diff
@@ -418,106 +332,75 @@ async def report(
             priority_fee=priority_fee,
             legacy_gas_price=legacy_gas_price,
             expected_profit=expected_profit,
-            chain_id=cid,
+            chain_id=core.config.main.chain_id,
             gas_price_speed=gas_price_speed,
             reporting_diva_protocol=reporting_diva_protocol,
             stake_amount=stake,
         )
 
         _ = input("Press [ENTER] to confirm settings.")
 
```

Comment:
No tellorX support?

Reply:
Do we need to be able to report to tellor X?

Reply:
@tkernell confirms we do not need to support tellor x reporting

Reply:
actually maybe not remove tellorflex script since arbitrum/optimisim are still not deployed

Reply:
Roger that, won't remove the tellor flex stuff. I don't think we need tellor x things though, will keep those removed. I changed my mind about the "leaving the code for people that want to report to the old contracts" idea. I don't think such ppl exist hah